parser.go

```diff
@@ -433,6 +443,17 @@ func (p *parser) parseInfo() error {
 	return nil
 }
 
+func parsePackageAliases(comment string) (*MapDefinition, error) {
+	re, _ := regexp.Compile("\"([^\"]*)\"")
```

Comment:
You can use `MustCompile` if you want to assert that there will be no error. I assume instead of returning an error it'll panic.

```suggestion
	re := regexp.MustCompile("\"([^\"]*)\"")
```